package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "markedit-aligned-table",
+  "version": "1.0.0",
+  "description": "",
+  "main": "lib/index.js",
+  "scripts": {
+    "test": "npm run lint; npm run unit",
+    "unit": "mocha -R spec",
+    "lint": "eslint ."
+  },
+  "keywords": [
+    "markdown",
+    "markdown-it",
+    "markdown-it-plugin"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/apiaryio/markedit-aligned-tables.git"
+  },
+  "author": "Apiary s.r.o.",
```

Comment:
You are right—although authoritative source for us is https://docs.apiary-internal.com/content/projects.html?highlight=project#creating-new-libraries-and-services :)